controller/instance_handler.go

```diff
@@ -166,6 +173,19 @@ func (h *InstanceHandler) getPod(podName string) (*v1.Pod, error) {
 	return h.pLister.Pods(h.namespace).Get(podName)
 }
 
+func (h *InstanceHandler) getPodLogs(podName string, taillines int) (string, error) {
+	tails := int64(taillines)
+	req := h.kubeClient.CoreV1().Pods(h.namespace).GetLogs(podName, &v1.PodLogOptions{
+		Timestamps: true,
+		TailLines:  &tails,
+	})
+	if req.URL().Path == "" {
+		return "", fmt.Errorf("rest.Request's Path is empty, it's a fake GetLogs ")
```

Comment:
Make the error message straightforward, so if we saw this in the production run, we can pinpoint the issue.

```
return "", fmt.Errorf("GetLogs for %v returns empty request path, may due to unit test run: %+v", podName, req).
```